src/vic3_world/world/vic3_world_importer.cpp

```diff
@@ -288,8 +290,8 @@ vic3::World vic3::ImportWorld(const configuration::Configuration& configuration)
    Log(LogLevel::Info) << "-> Reading Vic3 install.";
    commonItems::ModFilesystem mod_filesystem(fmt::format("{}/game", configuration.vic3_directory),
        mod_loader.getMods());
-   const auto province_definitions = LoadProvinceDefinitions(mod_filesystem);
-   const auto state_regions = ImportStateRegions(mod_filesystem);
```

Comment:
I draw the line at having to declare variables 3 separate times. (once in World, again in WorldOptions, and a third time in the world parser).
Changed parser to directly use WorldOptions so you only have to declare variables twice.

Reply:
Thanks! As you might have guessed, the parser predated WorldOptions and I forgot to clean everything up.